yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/column-statistics.psql

```diff
@@ -0,0 +1,25 @@
+CREATE TEMP TABLE column_statistics AS 
+    SELECT
+        st.schemaname as schema_name,
+        st.tablename as table_name,
+        st.attname as column_name,
+        st.null_frac as null_frequency,
+        CASE 
+            WHEN st.n_distinct < 0 THEN ROUND(-1 * st.n_distinct * c.reltuples)
+            ELSE st.n_distinct
+        END AS effective_n_distinct,
+        COALESCE((st.most_common_freqs::float8[])[1], 0) as max_frequency, --most_common_freqs is always sorted where maximum one appears first
+        (st.most_common_vals::text::text[])[1] as max_frequent_val --corresponding value is max_frequent_value
+    FROM pg_stats st
+    JOIN pg_class c ON c.relname = st.tablename
+    JOIN pg_namespace ns ON ns.nspname = st.schemaname AND ns.oid = c.relnamespace
+    WHERE st.schemaname=ANY(ARRAY[string_to_array(:'schema_list', '|')]) --schema_list
+    ORDER BY st.schemaname, st.tablename, st.attname;
```

Comment:
do we really need this order by?

Reply:
not really, this is just the data is readable if its sorted.